lib/puppet-lint/plugins/check_whitespace/no_colon_whitespace_after.rb

```diff
@@ -0,0 +1,25 @@
+# Public: Check the manifest tokens for resource type declaration that has no whitespace after a colon, and record a warning for each instance found.
+# #
+# https://docs.puppet.com/guides/style_guide.html#spacing-indentation-and-whitespace
```

Comment:
I'll go ahead and remove this test, it addresses a spacing implied but not written out.